recipes/pymodbus/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "pymodbus" %}
+{% set version = "2.3.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pymodbus-{{ version }}.tar.gz
+  sha256: 8e59d49cdc3f9df8e90fa004baa4aae94814b9ecf1fc03620b1d67234896c444
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - pymodbus.console=pymodbus.repl.main:main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - pyserial >=3.4
+    - python
+    - six >=1.11.0
+
+test:
+  imports:
+    - pymodbus
+    - pymodbus.client
+  requires:
+    - pip
```

Comment:
```suggestion
    - pip
  commands:
    - pymodbus.console --help
    - pip check
```

Reply:
Could you please tell why these additional commands are required?